docs/how-to/ci.rst

```diff
@@ -0,0 +1,160 @@
+Building your App in CI with GitHub Actions
+===========================================
+
+This GitHub `Actions <https://docs.github.com/en/actions>`__ workflow provides
+the basic framework necessary to test, build, and package a Briefcase project
+in CI for Windows, Linux, macOS, iOS, and Android.
+
+The same set of steps are used for each platform via a `matrix strategy
+<https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow>`__.
+This will create a separate job for each target platform with specific
+configuration.
```

Comment:
Could be explicit here about adding other platforms - "this sample packages for macOS, ...; if you wanted to add an Ubuntu 22.04 build, add another matrix entry " etc